src/kernel/src/fs/file.rs

```diff
@@ -75,8 +75,25 @@ impl Write for VFile {
 
 /// Type of [`VFile`].
 #[derive(Debug)]
+#[rustfmt::skip]
 pub enum VFileType {
-    Vnode(Arc<Vnode>), // DTYPE_VNODE
+    Vnode(Arc<Vnode>), // DTYPE_VNODE = 1
```

Comment:
@ultimaweapon Before you ask, yes, I did check all of these. Just look at every instance of 'finit' being called.